.github/workflows/create-release.yml

```
@@ -4,31 +4,35 @@ on:
   workflow_dispatch:
     inputs:
       name:
-        description: 'Release name'
+        description: 'Release name ( e.g. "2.1.3" )'
         default: ""
         required: true
 
 jobs:
   verify-head-status:
-    name: Verify head
+    name: Verify head (image version and prow job)
     runs-on: ubuntu-latest
 
     steps:
+      - name: Verify that the current branch has a name that starts with 'release-'
+        run: |
+          CURRENT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
+          if [[ "$CURRENT_BRANCH" == release-* ]]; then
+            echo "Branch name starts with 'release-'."
+          else
+            echo "Branch name does not start with 'release-'."
+            exit 1
+          fi  
       - name: Checkout code
         uses: actions/checkout@v4
         with:
           fetch-depth: 0
```

```suggestion
          fetch-depth: 0
          
```
let's add some whitespace for consistent formatting.